### PR TITLE
Updating SetupHelp.md 

### PR DESCRIPTION
**Pull request by SamuilsRulovs**
Changes made:
- Added more description to "Boost 1_65_1" installation; 
- Added Information about getting Dot and Doxygen

### CODE REVIEW

File: Documentation/markdown/SetupHelp.md [hunk 1]
```diff
@@ -24,7 +24,11 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
 
 		* **HINT:** There are components, which Qt Online Installer selects as default options. You can uncheck these components for saving computer memory.
 
-4. **Boost 1_65_1** is installed (find [here](https://sourceforge.net/projects/boost/files/boost-binaries/1.65.1/boost_1_65_1-msvc-14.1-64.exe/download)) and a new environment variable called Boost_INCLUDE_DIR is created. This variable should point to the binary folder, where **Boost 1_65_1** is staged (e.g. C:\local\boost_1_65_1\).
+4. **Boost 1_65_1** is installed (find [here](https://sourceforge.net/projects/boost/files/boost-binaries/1.65.1/boost_1_65_1-msvc-14.1-64.exe/download)).
+
+	* Create a folder named thirdparty in C:\ and install **Boost 1_65_1** to C:\thirdparty\vs2015\x64\boost_1_65_1.
```
Comment (pjanck) [inline Documentation/markdown/SetupHelp.md]:
```suggestion
	* Create a folder named thirdparty in `C:\` and install **Boost 1_65_1** to `C:\thirdparty\vs2015\x64\boost_1_65_1`.
```
Please use markdown's formating for code snippets.

File: Documentation/markdown/SetupHelp.md [hunk 2]
```diff
@@ -24,7 +24,11 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
...
+	* Add this path into the environment variables. (Create new environment variable called Boost_INCLUDE_DIR. This variable should point to the binary folder, where **Boost 1_65_1** is staged (e.g. C:\thirdparty\vs2015\x64\boost_1_65_1).
```
Comment (pjanck) [inline Documentation/markdown/SetupHelp.md]:
```suggestion
	* Create new environment variable called `Boost_INCLUDE_DIR`. This variable should point to the binary folder, where **Boost 1_65_1** is staged (e.g. `C:\thirdparty\vs2015\x64\boost_1_65_1`).
```

File: Documentation/markdown/SetupHelp.md [hunk 3]
```diff
@@ -35,6 +39,14 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
...
  Clone **OpenInfraPlatform** repository  - more information you can find [here](./GitProcess.md)
 
+ Get Dot and Doxygen:
+
+	 1. After cloning **OpenInfraPlatform** repository open OpenInfraPlatform source folder. 
+	 2. Open folder **external**
+	 3. Open files:
+		* GetDot.cmd  
+		* GetDoxygen.cmd
+
```
Comment (pjanck) [inline Documentation/markdown/SetupHelp.md]: This belongs in a separate section, called "Generating documentation" at the end of the document.

**Review by pjanck — CHANGES_REQUESTED** [review]
Small changes, but generally great work! 

After addressing these, the pull request can be merged. Does it address any issues? Please provide their numbers.

File: Documentation/markdown/SetupHelp.md (hunk 2 above)
Comment (SamuilsRulovs) [reply Documentation/markdown/SetupHelp.md:30]: Resolved: 77865a7

File: Documentation/markdown/SetupHelp.md (hunk 3 above)
Comment (SamuilsRulovs) [reply Documentation/markdown/SetupHelp.md]: Resolved: 439c61b

File: Documentation/markdown/SetupHelp.md (hunk 1 above)
Comment (SamuilsRulovs) [reply Documentation/markdown/SetupHelp.md]: Resolved: 77865a7

File: Documentation/markdown/SetupHelp.md (hunk 2 above)
Comment (SamuilsRulovs) [reply Documentation/markdown/SetupHelp.md]: Resolved:  31e2f94

**Comment by SamuilsRulovs** [comment]
> Small changes, but generally great work!
> 
> After addressing these, the pull request can be merged. Does it address any issues? Please provide their numbers.

Issue numbers:  #128 & #129

File: Documentation/markdown/SetupHelp.md [hunk 4]
```diff
@@ -86,4 +109,15 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
...
+	**NOTE:** execute **Get_OKLABI.cmd** in the source folder (e.g. `C:\dev\Open-Infra-Platform\external`) to prevent an error during the build process. After the execution of **Get_OKLABI.cmd**, it's necessary to *configure* and *generate* OpenInfraPlatform project once again in CMake.
+
+
+## <a name="Generating_Doc"></a> Generating documentation
+
+### <a name="Get_Dot"></a> Get Dot and Doxygen
+
+1. After cloning **OpenInfraPlatform** repository open OpenInfraPlatform source folder. 
+2. Open folder **external**
+3. Open files:
```
Comment (pjanck) [inline Documentation/markdown/SetupHelp.md]:
```suggestion
3. Execute:
```

File: Documentation/markdown/SetupHelp.md [hunk 5]
```diff
@@ -86,4 +109,15 @@ Before you can proceed to the Installation of **OpenInfraPlatform**, check, if t
...
+	* GetDot.cmd  
+	* GetDoxygen.cmd
```
Comment (pjanck) [inline Documentation/markdown/SetupHelp.md]: Add a reference/link to Doxygen help document after this line, sth like "Read more about Doxygen in our guidelines".

Comment (pjanck) [reply Documentation/markdown/SetupHelp.md]: Also, always end any file within our repository with an empty line (that is, the last line needs to be empty).

**Review by pjanck — CHANGES_REQUESTED** [review]
Small tweaks at the end, otherwise it's ripe to merge.

File: Documentation/markdown/SetupHelp.md (hunk 4 above)
Comment (SamuilsRulovs) [reply Documentation/markdown/SetupHelp.md]: Resolved:  a4a7d9c

File: Documentation/markdown/SetupHelp.md (hunk 5 above)
Comment (SamuilsRulovs) [reply Documentation/markdown/SetupHelp.md]: Resolved:  a4a7d9c

**Review by pjanck — APPROVED** [review]
Thank you. This improves our documentation nicely.

**Comment by pjanck** [comment]
Fixes #128 , fixes #129 .

**Pull request merged**